core/src/services/s3/backend.rs

```diff
@@ -304,6 +304,8 @@ pub struct S3Builder {
 
     http_client: Option<HttpClient>,
     customed_credential_load: Option<Box<dyn AwsCredentialLoad>>,
+
+    write_buffer_size: Option<usize>,
```

Comment:
Please call this value `write_min_size`.

Reply:
ditto.